gnovm/pkg/gnolang/realm.go

```diff
@@ -651,7 +665,12 @@ func (rlm *Realm) markDirtyAncestors(store Store) {
 				// via .updated.
 				break
 			} else {
-				rlm.MarkDirty(po)
+				if po.GetIsDeleted() {
```

Comment:
maybe just make it another `else if` in the above chain?

Reply:
yes, 👍